src/test/java/meditracker/QuerySpecificMedication.java

```diff
@@ -0,0 +1,28 @@
+package meditracker;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+class QuerySpecificMedication {
+    /**
+     * Queries the array list containing the medication data
+     */
+    @Test
+    public void queryMedication_ifContains_expect() {
+        String inputString = "Omeprazole";
+
+        ArrayList<String> medications = new ArrayList<String>();
```

Comment:
Might be better to use type inference and also use interface instead of explicit type of list

```suggestion
        List<String> medications = new ArrayList<>();
```

Reply:
Got it. Thank you.